webbeans-impl/src/main/resources/META-INF/openwebbeans/openwebbeans.properties

```diff
@@ -220,6 +220,7 @@ org.apache.webbeans.scanExclusionPaths=/jre/lib, \
         /jsp-api, \
         /myfaces-api, \
         /myfaces-impl, \
+        /jakarta.faces, \
```

Comment:
shouldn't it be jakarta as "javax" is?

Reply:
I thought these libs were there maven coordinates which is...
```xml
                <dependency>
                    <groupId>org.glassfish</groupId>
                    <artifactId>jakarta.faces</artifactId>
                    <version>2.3.16</version>
                </dependency>
```

So I thought my choice above was the safest exclude.

Reply:
I am not sure who added `javax` here but it seems wrong to me.  and this problem with Mojarra only started with the GlassFish JARs when they got renamed to jakarta.faces.  the old javax versions of these JARS do not expose the issue.  I just know the above exclude definitely fixes the problem in OWB-1298.